### ISSUE TITLE
Screen reader does not announce when new/updated annotations are available

### ISSUE THREAD

**Issue by mattdricker** [hypothesis/product-backlog#1422]
This feedback comes out of Accessibility Web Audit from UC Berkeley.

### Current behavior
When new annotations are posted to the document/group currently loaded, Hypothesis shows an "updates available" button at the top of the sidebar. However, the appearance of this button is not announced by the screen reader.
<img width="513" alt="Screen Shot 2023-02-22 at 10 15 08 AM" src="https://user-images.githubusercontent.com/71603629/220700517-a5bf8856-3657-4e1c-9d8e-09e9f6ee67ae.png">

### Expected behavior
Screen reader will announce that there are new/updated annotations available to be loaded.

This is the feedback from the review:

> Screen reader did not know that the screen was updated.
>
>Use polite alert.

This was additional feedback that I believe is no longer an issue:
> Icon looks like a download or power button (“power download”).The button has a title attribute but those do not help keyboard-only users.
>
>Use visible text such as “New.”

**Comment by mattdricker** [comment]
@acelaya awesome work getting this landed! I heard through the grapevine that we've now also got a new keyboard shortcut to load the new annotations 🎉 

So I can best get this written up for the end-user who raised this issue, can you quickly tl;dr (or point me to the best existing summary) of how everything has come together and any specifics on the kb shortcut.

**Comment by acelaya** [comment]
Sure! (Actually I intended to share this on our changelog channel, but I got distracted with other tasks).

The testing steps described on [this PR](https://github.com/hypothesis/client/pull/5326) more or less cover the whole flow, but to summarize, we have done the next improvements:

* Screen readers announce the availability of new annotations as soon as possible (at the same time the button appears on screen).
* There's a new `l` keyboard shortcut ("l" after "load") which does the same as clicking on the button, loading the new annotations.
* Due to technology limitations, this shortcut only works while the sidebar is focused. Because of this, we delay a second screen reader notification which announces what's the shortcut and what will happen when used. This notifications is pushed as soon as the sidebar is focused, or imediatelly if it's already focused.

Let me know if that clarifies it.

**Comment by mattdricker** [comment]
This is perfect. Thank you!
